eldritch/events.py

```diff
@@ -2619,11 +2619,17 @@ def resolve(self, state):
         return False
 
     if self.seal_choice is None:
-      if self.character.clues < 5:  # TODO: this can also have modifiers
+      difficulty = 5
+      difficulty += state.ancient_one.get_modifier(self, "difficulty")
+      difficulty += state.get_modifier(self, "difficulty")
```

Comment:
You have double-counted here a bit - when you call `state.get_modifier`, it will get the modifier from all globals. The globals include: the environment, the rumor, the ancient one, and any other global effects. You do not need to list them individually here.

Reply:
Done.